packages/common/src/types/index.ts

```diff
@@ -57,6 +57,27 @@ export type Directory = {
   sourceId: string;
 };
 
+export type Template = {
```

Comment:
Don't we export this somewhere else? I think in `common/src/templates.ts` from the class.

Reply:
The one there is differnt